keras2onnx/common/onnx_ops.py

```diff
@@ -5,3 +5,19 @@
 ###############################################################################
 
 from onnxconverter_common.onnx_ops import *
+from onnxconverter_common.onnx_ops import _create_name_or_use_existing_one
+
+def apply_thresholded_relu(scope, input_name, output_name, container, operator_name=None, alpha=None):
```

Comment:
why not using apply_thresholded_relu in onnxconverter_common.onnx_ops ?

Reply:
The CI uses the published package version of onnxconverter_common, so it will have to wait for the new PyPi publish to use that version of the op. Does it make sense to wait for that instead?

Reply:
It's better to let onnxconverter_common publish first, otherwise we will change this code again. Is there any other change pending? Otherwise we can publish it.

Reply:
Now that onnxconverter-common is published, this code in onnx_ops has been removed.